src/components/BrowsePageComponent.tsx

```diff
@@ -1,20 +1,22 @@
+import { Auth } from "aws-amplify";
 import React, { useCallback, useEffect } from "react";
 import { useContext } from "react";
 import CollectionStore from "../Database/CollectionStore";
 import { CollectionModel } from "../domain/CollectionModel";
 import { useState } from "react";
 import "./PageLayoutComponent.css";
-import { CollectionBox } from "./CollectionBox";
+import { FakeCollectionBox } from "./FakeCollectionBox";
 import CollectRImage from "../images/CollectR.jpg";
 import { Link } from "react-router-dom";
 import FakeCollectionStore from "./FakeCollectionStore";
```

Comment:
I see there is a FakeCollection.  What is that being used for?

Reply:
A better word would be "mock" but those are the collections displayed on the "Browse" page as a sample of what the user could search for!